src/test/groovy/de/benediktritter/maven/plugin/development/MavenEndToEndFuncTest.groovy

```diff
@@ -65,7 +65,7 @@ class MavenEndToEndFuncTest extends AbstractPluginFuncTest {
         def mavenBuildResult = mvnCleanPackage(mavenBuild)
 
         then:
-        mavenBuildResult.contains("touch-maven-plugin:1.0.0:touch")
+        mavenBuildResult.contains("[INFO] --- touch:1.0.0:touch (default) @ use-touch-plugin ---")
```

Comment:
Looks like the output here changed in one of the more recent versions of Maven. The new code works on my machine but fails on GitHub runners. Reverting to the version that works on GitHub for now.

This needs a solution where the test defines the version of maven to use and downloads that to a temporary location.

```suggestion
        mavenBuildResult.contains("touch-maven-plugin:1.0.0:touch")
```